src/download-package.js

```diff
@@ -18,20 +26,22 @@ async function downloadPackage(name, url, range) {
 
   let newTmpDir = await createTmpDir();
   let { stdout } = await spawn('npm', ['install', url], { cwd: newTmpDir });
-  // if (!name) {
```

Comment:
Can we keep these comments in for future me please :)

Reply:
gotcha, reverted